packages/core/src/mongoMeasurement.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  MongoTracer,
+  StartInstrumentsRequestBody,
+  StartInstrumentsResponse,
+} from "./generated/openapi";
+
+export type { StartInstrumentsRequestBody };
+
+export class MongoMeasurement {
+  private tracerClient: MongoTracer | undefined;
+
+  constructor() {
+    const serverUrl = process.env.CODSPEED_MONGO_INSTR_SERVER_ADDRESS;
+
+    if (serverUrl !== undefined) {
+      this.tracerClient = new MongoTracer({
+        BASE: serverUrl,
+      });
+    }
+  }
+
+  public async startInstruments(
```

Comment:
```suggestion
  public async setup(
```

Would be clearer. Renaming it as well in mongo-tracer would be nice.